src/Microsoft.AspNetCore.Mvc.Core/ApplicationModels/ControllerModel.cs

```diff
@@ -98,6 +100,22 @@ public ControllerModel(ControllerModel other)
 
         public IList<IFilterMetadata> Filters { get; }
 
+        // For unit testing.
+        internal IReadOnlyList<object> ControllerAssemblyAttributes
```

Comment:
What''s the purpose of this?

Reply:
Unit testing. I don't want to decorate the test assembly with attributes